gulpfile.js

```diff
@@ -26,16 +27,17 @@ gulp.task("compile", () => {
 
   const tsStream = tsResult.js
     .pipe(babel())
+    .pipe(replace("../src/__generated__", "./__generated__")) // FIXME: Remove once Relay updates have been made
```

Comment:
This is the key change. Due to how https://github.com/artsy/reaction/blob/master/.babelrc#L4 encodes the full path, this will make that relative to output.